jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenterTest.java

```diff
@@ -565,4 +564,62 @@ public void testStatusSettingsColumns(){
                 COLUMN_PROCESS_INSTANCE_DESCRIPTION
         );
     }
+
+    @Test
+    public void testCompletedTaskSelection() {
+        TaskSummary taskSummary = TaskSummary.builder()
+                .id(TASK_ID)
+                .deploymentId(TASK_DEPLOYMENT_ID)
+                .status(TASK_STATUS_COMPLETED)
+                .build();
+        boolean closed = true;
+        when(placeManager.getStatus(any(DefaultPlaceRequest.class))).thenReturn(PlaceStatus.CLOSE);
+
+        getPresenter().selectTask(taskSummary,
+                                  closed);
+
+        verify(placeManager).goTo(TaskDetailsMultiPresenter.SCREEN_ID);
+        final ArgumentCaptor<TaskSelectionEvent> captor = ArgumentCaptor.forClass(TaskSelectionEvent.class);
+        verify(taskSelected).fire(captor.capture());
+        assertTrue(captor.getValue().isForLog());
+    }
+
+    @Test
+    public void testExitedTaskSelection() {
+        TaskSummary taskSummary = TaskSummary.builder()
+                .id(TASK_ID)
+                .deploymentId(TASK_DEPLOYMENT_ID)
+                .status(TASK_STATUS_EXITED)
+                .build();
+        boolean closed = true;
+        when(placeManager.getStatus(any(DefaultPlaceRequest.class))).thenReturn(PlaceStatus.CLOSE);
+
+        getPresenter().selectTask(taskSummary,
+                                  closed);
+
+        verify(placeManager).goTo(TaskDetailsMultiPresenter.SCREEN_ID);
+        final ArgumentCaptor<TaskSelectionEvent> captor = ArgumentCaptor.forClass(TaskSelectionEvent.class);
+        verify(taskSelected).fire(captor.capture());
+        assertTrue(captor.getValue().isForLog());
+    }
```

Comment:
Maybe we don't need to tests both read-only statuses separately. I think testing only one won't reduce test coverage in any way.